src/components/Settings.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactElement, useContext} from "react";
+import { useNavigate } from 'react-router-dom';
+import { LoginContext } from "../App";
+
+const Settings = (): ReactElement => {
+
+    const navigate = useNavigate();
+    const { setUser } = useContext(LoginContext); 
+
+    function LogOut() {
```

Comment:
With functions, I would name them with camel case convention (i.e. logOut or logout depending on whether you think it should be one word lol)